README.md

```diff
@@ -90,19 +90,25 @@ corrections in tag values or units.
 Here is a summary of the types of changes that correspond to different
 levels of changes in the semantic version:
 
-| Change                          | Semantic-level | 
-| ---------------------------------- | -------------- |
-| Major addition to HED functionality     | Major  |
-| Tag deleted from schema.                | Major  |
-| Unit or unit class removed from node.   | Major  |
-| New tag added to the schema.            | Minor  |
-| New attribute added to schema.          | Minor  |
-| New unit class or unit added to schema. | Minor  |
-| New unit class added to node.           | Minor  |
-| Node moved in schema without change in meaning. | Minor |
-| Revision of description field in schema.        | Patch   |
-| Correction of suggestedTag or relatedTag.       | Patch  |
-| Correction of wiki syntax such as closing tags. | Patch |
+| Change                                              | Semantic-level |
```

Comment:
Add the following at the end of the previous paragraph:

Major version changes to the schema may be backwards incompatible and thus impact existing annotations. Guidance on how existing annotations should be updated will be provided with major schema version releases.